### ISSUE TITLE
Problem with nastring for non-numeric columns

### ISSUE THREAD

**Issue by bkamins** [queryverse/CSVFiles.jl#31]
Because the default `nastring` is `NA` there is a following problem:
1. take a data structure that has e.g. `String` column with missing data in it;
2. `save` it to disk using default parameters; missings get converted to `NA` on disk
3. `load` it back and you have `"NA"` string where you earlier had missings

The same problem occurs with e.g. `Char` data.

While `NA` is a sensible default for numeric columns it is a bit confusing for non-numeric columns (and actually can lead to wrong results as it is fully possible to have `NA` string in data).

I think that it would be best to have an empty string for missings in non-numeric data.

**Comment by davidanthoff** [comment]
> I think that it would be best to have an empty string for missings in non-numeric data.

That would be for writing files, right? Do you think we need to also change something about reading?

**Comment by bkamins** [comment]
Frankly - for reading I would never create a missing when reading a `String` but leave as is and let the user decide what to do.
It is perfectly possible that `"NA"` sting means something and is present if a CSV file. E.g. in Polish this is a valid word.

A second best solution would be to treat empty string as missing (although I can imagine situations where `""` might mean something, e.g. it is perfectly valid to have the following vector in Julia `["", missing]`, but at least it is not that problematic).

However, I realize that all this is breaking so please decide what you think is best in the context of whole queryverse.

**Comment by bkamins** [comment]
Good point - if all is quoted and only `NA` is unquoted this a clear way do distinguish it. This is what `write.csv` in R does (although then `read.csv` reads back both of them as missing 😄).
